doc/sphinx/automatic-tactics.rst

```diff
@@ -0,0 +1,20 @@
+==================================
+Automatic and programmable tactics
+==================================
+
+Some tactics are largely automated and are able to solve complex
```

Comment:
Can we be more precise than "automated"?  I'd say Ltac is also automated, or allows automation.

Reply:
I'll try, but there is a blurry line between a programmable tactic and a tactic language.

Reply:
I don't know what precisely to clarify here. Ltac allows to write automatic tactics, yes. But this chapter presents *built-in* automated or programmable tactics.

Reply:
Then how about adding "Built-in" to the chapter title?

> but there is a blurry line between a programmable tactic and a tactic language

How do you distinguish the two?  Offhand, they seem almost synonymous.

Reply:
While there are things you can do with one or the other, a language gives you basic constructs to combine into complex expressions while a programmable tactic only gives you extension points.

Reply:
Still not entirely getting you.  Extension points meaning something you change by writing a plugin?  Not familiar with the Coq meaning of "extension points".

Reply:
I used this as a generic term, not as a Coq-specific one. These programmable tactics usually provide dedicated commands (such as the `Hint` family of commands) to extend the default behavior of the tactic.